src/main/java/com/google/step/snippet/external/W3SchoolClient.java

```diff
@@ -6,12 +6,23 @@
 import org.jsoup.nodes.Document;
 import org.jsoup.select.Elements;
 
-public final class W3SchoolClient {
+public final class W3SchoolClient implements Client {
   private static final String TITLE_TAG = "h1";
   private static final String DESC_TAG = "p";
   private static final String SNIPPET_CLASS = "w3-example";
   private static final String CODE_CLASS = "w3-code";
+  private final String cseId;
 
+  public W3SchoolClient(String cseId) {
+    this.cseId = cseId;
+  }
+
+  @Override
+  public String getCseId() {
+    return cseId;
+  }
+
+  @Override
```

Comment:
Add a JavaDoc for this client, similar to the one added for StackOverflowClient.